src/styles/_catalog.scss

```diff
@@ -1,22 +1,41 @@
 .catalog {
-  display: flex;
+  display: grid;
   flex-wrap: wrap;
 
   max-width: 944px;
   margin: 0 auto;
+  gap: 48px 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
+    display: grid;
+    place-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 487px) {
```

Comment:
You use this grid-template-columns: repeat(1, minmax(200px, 1fr));  without media query. when requests will be triggered this rule will be overwritten